webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -565,29 +568,6 @@ export default class SuggestionBox extends React.PureComponent {
         return this.state.items.some((item) => !item.loading);
     };
 
-    confirmPretext = () => {
-        const textbox = this.getTextbox();
-        const pretext = textbox.value.substring(0, textbox.selectionEnd).toLowerCase();
-
-        if (this.pretext !== pretext) {
-            this.handlePretextChanged(pretext);
-        }
-    };
-
-    handleKeyUp = (e) => {
-        this.confirmPretext();
-        if (this.props.onKeyUp) {
-            this.props.onKeyUp(e);
-        }
-    };
-
-    handleMouseUp = (e) => {
```

Comment:
Why we no longer need the mouse and key events?

Reply:
We pass `{...props}` through to the `QuickInput` below, so the props are still being passed through as they were before. The `SuggestionBox` doesn't care about them itself any more